docs/template_plugin/tests/functional/op_reference/base_reference_test.hpp

```diff
@@ -67,6 +85,11 @@ struct Tensor {
     Tensor(const ngraph::Shape& shape, ngraph::element::Type type, const std::vector<T>& data_elements)
         : Tensor {shape, type, CreateBlob(type, data_elements)} {}
 
+    // Use this constructor of dynamic network inputs
+    template <typename T>
+    Tensor(ngraph::element::Type type, const ngraph::Shape& shape, const std::vector<T>& data_elements)
```

Comment:
Why do we need to have new constructor with different order of arguments?

Reply:
In these tests blob is created based on input shape. However for 'resize' the common case is that width&height are dynamic. So, test blob should be created based on 'input data elements shape', not based on 'parameter shape'
I didn't find easier solution, so decided to introduce one more constructor  and one more variant of 'CreateBlob' with specified shape

Reply:
I also don't understand why do we need new ctor..

Reply:
After rebase, still had to introduce one more constructor as modifying this causes a lot of unexpected failures of other tests